client/src/libs/jquery/jquery.wymeditor.js

```diff
@@ -317,380 +348,390 @@ jQuery.extend(WYMeditor, {
-               + "</body>",
-
-    dialogPasteHtml:  "<body class='wym_dialog wym_dialog_paste'"
-               + " onload='WYMeditor.INIT_DIALOG(" + WYMeditor.INDEX + ")'"
-               + ">"
-               + "<form>"
-               + "<input type='hidden' class='wym_dialog_type' value='"
-               + WYMeditor.DIALOG_PASTE
-               + "' />"
-               + "<fieldset>"
-               + "<legend>{Paste_From_Word}</legend>"
-               + "<div class='row'>"
-               + "<textarea class='wym_text' rows='10' cols='50'></textarea>"
-               + "</div>"
-               + "<div class='row'>"
-               + "<input class='wym_submit' type='button'"
-               + " value='{Submit}' />"
-               + "<input class='wym_cancel' type='button'"
-               + "value='{Cancel}' />"
-               + "</div>"
-               + "</fieldset>"
-               + "</form>"
-               + "</body>",
-
-    dialogPreviewHtml: "<body class='wym_dialog wym_dialog_preview'"
-                      + " onload='WYMeditor.INIT_DIALOG(" + WYMeditor.INDEX + ")'"
-                      + "></body>",
-                      
-    dialogStyles: [],
-
-    stringDelimiterLeft: "{",
-    stringDelimiterRight:"}",
-    
-    preInit: null,
-    preBind: null,
-    postInit: null,
-    
-    preInitDialog: null,
-    postInitDialog: null
-
-  }, options);
-
-  return this.each(function() {
-
-    new WYMeditor.editor(jQuery(this),options);
-  });
+jQuery.fn.wymeditor = function (options) {
+    options = jQuery.extend(
+        {
+            html: "",
+
+            basePath: false,
+
+            skinPath: false,
+
+            wymPath: false,
+
+            iframeBasePath: false,
+
+            jQueryPath: false,
+
+            styles: false,
+
+            stylesheet: false,
+
+            skin: "default",
+            initSkin: true,
+            loadSkin: true,
+
+            lang: "en",
+
+            direction: "ltr",
+
+            boxHtml:
+                "<div class='wym_box'>" +
+                "<div class='wym_area_top'>" +
+                WYMeditor.TOOLS +
+                "</div>" +
+                "<div class='wym_area_left'></div>" +
+                "<div class='wym_area_right'>" +
+                WYMeditor.CONTAINERS +
+                WYMeditor.CLASSES +
+                "</div>" +
+                "<div class='wym_area_main'>" +
+                WYMeditor.HTML +
+                WYMeditor.IFRAME +
+                WYMeditor.STATUS +
+                "</div>" +
+                "<div class='wym_area_bottom'>" +
+                WYMeditor.LOGO +
+                "</div>" +
+                "</div>",
+
+            logoHtml: "<a class='wym_wymeditor_link' " + "href='http://www.wymeditor.org/'>WYMeditor</a>",
+
+            iframeHtml:
+                "<div class='wym_iframe wym_section'>" +
+                "<iframe " +
+                "src='" +
+                "get_editor_iframe' " +
```

Comment:
@dannon this was the change I had to make. I could just do this without prettier, or is that not looking like the right fix ?